frontend/src/pages/ErrorTags/ErrorTagsContainer.tsx

```diff
@@ -59,7 +58,6 @@ export function ErrorTagsContainer() {
 						minHeight: '100%',
 						width: '100%',
 					}}
-					className={containerContentStyles}
```

Comment:
cc @deltaepsilon per this [comment](https://highlightcorp.slack.com/archives/C02CJANPHQS/p1693604331085109?thread_ts=1693603340.257949&cid=C02CJANPHQS).